compiler/src/iree/compiler/Codegen/Common/MaterializeEncodingIntoPackUnPack.cpp

```diff
@@ -901,6 +911,18 @@ class MaterializeContractionOp
 
     auto converter = static_cast<const MaterializeEncodingTypeConverter *>(
         this->getTypeConverter());
+
+    if (auto layoutAttr = converter->getLayoutAttr()) {
+      SmallVector<Type> convertedResTypes;
+      for (auto init : op.getDpsInits()) {
+        convertedResTypes.push_back(converter->convertType(init.getType()));
+      }
+      SmallVector<Operation *> newOps =
+          layoutAttr.lowerComputeOp(rewriter, op, convertedResTypes, operands);
+      rewriter.replaceOp(op, newOps.back());
```

Comment:
Talking about this call site, in my other review comment about lowerComputeOp. Why `.back()` and not another of the potentially multiple elements in the `newOps` vector --- or if this is hardcoding that `newOps` only has one element, then why is it a vector?